packages/components/popper/src/content.ts

```diff
@@ -103,3 +104,5 @@ export type UsePopperContentProps = ExtractPropTypes<
 export type UsePopperCoreConfigProps = ExtractPropTypes<
   typeof usePopperCoreConfigProps
 >
+
+export type PopperContentInstance = InstanceType<typeof Content>
```

Comment:
Hello! if you look at the component/popper/index.ts file, there is a source code that performs the same role as this syntax.

looks like one of the two needs to be fixed.


```ts
import { withInstall } from '@element-plus/utils'
import Popper from './src/popper.vue'

import ElPopperArrow from './src/arrow.vue'
import ElPopperTrigger from './src/trigger.vue'
import ElPopperContent from './src/content.vue'

export { ElPopperArrow, ElPopperTrigger, ElPopperContent }

export const ElPopper = withInstall(Popper)
export default ElPopper

export * from './src/popper'
export * from './src/trigger'
export * from './src/content'
export * from './src/arrow'

export type { Placement, Options } from '@popperjs/core'
export type ElPopperArrowInstance = InstanceType<typeof ElPopperArrow> // here
export type ElPopperArrowTrigger = InstanceType<typeof ElPopperTrigger> // here
export type ElPopperArrowContent = InstanceType<typeof ElPopperContent> // here
```

Reply:
Thanks for comments, I have refactor it in another [PR](https://github.com/element-plus/element-plus/pull/8529)